packages/playground/blueprints/bin/generate-schema.js

```diff
@@ -41,7 +41,7 @@ schema.definitions.Blueprint.properties.$schema = {
 	type: 'string',
 };
 
-// Use a discriminator to help the Ajv JSON schema validator
+// Use a discriminator to help the jsonschema JSON schema validator
```

Comment:
This may be misleading because jsonschema does not appear to support `discriminator`:
https://github.com/search?q=repo%3Atdegrunt%2Fjsonschema+discriminator&type=code